### PR TITLE
Allow for multiple root directories when scanning for fence files

### PR DESCRIPTION
**Pull request by Adjective-Object**
## Motivation

In owa, we currently rely on excluding the node_modules directory.

This works fine under current usage, but BuildXL by convention stores state in the Out directory, which stores unique logs across multiple runs -- this was causing good-fences to take upwards of 20 minutes on subsequent runs.

By specifying the rootDirs as `packages` and `shared` instead of relying on traversing from the common directory `.`, we speed up good-fences (and avoid transient read errors caused by files being deleted out from under good-fences as it traverses BuildXL's internal state).

## Changes
- changes rootDir to an array of strings
- supports falling back to rootDir: string as an argument for backwords compatibility
- upgrades commander to use variadic arguments to support rootDir from the cli

### CODE REVIEW

**Comment by smikula** [comment]
@Adjective-Object Is there a reason to prefer multiple root directories as opposed to adding an option to exclude directories?  The latter seems more intuitive to me, and more in line with something like `.gitignore` or `.prettierignore`.

I'm imagining an optional `ignoreDirs` option (`string[]`) that — in order to be backward compatible — gets autopopulated with `node_modules` based on the `ignoreExternalFences` setting.  Longer term supporting something like a `.fencesignore` file would be great.

**Comment by Adjective-Object** [comment]
Do you feel strongly about having an ignore file over a list of includes? As much as I like the aesthetics of .ignore files:
- There's no ecosystem coherence, Jest has a list of roots, prettier has an .ignorefile, tsconfig has both with the more commonly used in tutorials being "include"/
- Ignorefiles come with the expectation of git pattern matching, which means pulling in [ignore](https://www.npmjs.com/package/ignore), as well as supplying a way to set the ignore file from the command line.
- Ignorefiles also mean performing a directory tree traversal to find ignorefiles when not run in a repo root. Which means our FS walk has to to worry about symlink loops, and cross-fs boundaries.
- As you develop you are more likely to introduce more roots you want to ignore than roots you want to pay attention to (as is the case here). Moreover those new roots don't immediately cause failures, but rather cause slow performance degradation.
- OWA is the main consumer of good-fences, and rewriting for aesthetics while introducing more complexities doesn't seem productive to me.

File: src/utils/getOptions.ts [hunk 1]
```diff
@@ -1,6 +1,7 @@
-import RawOptions from '../types/RawOptions';
-import Options from '../types/Options';
-import normalizePath from './normalizePath';
+import RawOptions from "../types/RawOptions";
```
Comment (smikula) [inline src/utils/getOptions.ts:1]: Looks like I don't have prettier configured in this repo yet.  For my sanity, can you use single quotes for all the string literals in this file?

Comment (Adjective-Object) [reply src/utils/getOptions.ts:1]: Looks like you beat me to it!

**Review by smikula — APPROVED** [review]

**Pull request merged**